pkg/cmd/cli/cmd/export.go

```diff
@@ -99,7 +101,7 @@ func RunExport(f *clientcmd.Factory, exporter Exporter, in io.Reader, out io.Wri
 
 	mapper, typer := f.Object()
 	b := resource.NewBuilder(mapper, typer, f.ClientMapperForCommand()).
-		NamespaceParam(cmdNamespace).DefaultNamespace().
+		NamespaceParam(cmdNamespace).DefaultNamespace().AllNamespaces(allNamespaces).
 		FilenameParam(explicit, filenames...).
 		SelectorParam(selector).
 		ResourceTypeOrNameArgs(all, args...).
```

Comment:
@bparees That would require a change here.  If you passed `true` in place of `all`, you'd get the "normal" behavior you get from `oc get`.  

@smarterclayton what say you?